test/common.js

```diff
@@ -51,19 +51,17 @@ function rmdirSync(p, originalEr) {
   }
 }
 
-function refreshTmpDir() {
-  if (!process.send) { // Not a child process
-    try {
-      rimrafSync(exports.tmpDir);
-    } catch (e) {
-    }
+exports.refreshTmpDir = function() {
+  try {
+    rimrafSync(exports.tmpDir);
+  } catch (e) {
+  }
```

Comment:
`make jslint` doesn't complain. There's one similar occurrence in the file (although it has a comment in the empty block so maybe that's different): https://github.com/nodejs/io.js/blob/a6b8ee19b85bbd798510191f0aee596f36b909d2/test/common.js#L159-L161